src/containers/Secrets/Secrets.js

```diff
@@ -60,19 +61,23 @@ export /* istanbul ignore next */ class Secrets extends Component {
   handleDeleteSecretClick = value => {
     this.setState({
       openDeleteSecret: true,
-      toBeDeleted: value
+      toBeDeleted: value.map(secret => ({
+        namespace: secret.id.split(':')[0],
```

Comment:
rather than doing this string manipulation these values should be provided as an object, I thought that was done in a previous PR